### PR TITLE
Ana/fix micro denoms

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Together with my other PR for the API, #255 , finishes fixing (for now) the issues we had in the action modals for sending transactions and staking in other networks apart from Cosmos.

It just add the other networks micro denoms in a dictionary. Thanks @faboweb for the "complete" hint :wink: .

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
hahahaha thanks @Bitcoinera for the completion

**Comment by Bitcoinera** [comment]
You are very welcome!

But hey, what about the tests? :woman_shrugging: 
I am going to make another PR then for them. I know, not so much to test, but still...

**Comment by faboweb** [comment]
sure! the code you added was covered already though.

File: src/scripts/common.js [hunk 1]
```diff
@@ -45,8 +45,11 @@ export const sleep = function(amount) {
 }
 
 export function toMicroDenom(denom) {
-  if (denom === "ATOM") {
-    return "uatom"
+  const lookup = {
+    ATOM: "uatom",
```
Comment (jbibla) [inline src/scripts/common.js]: will we need to add this for every network?

Comment (Bitcoinera) [reply src/scripts/common.js]: I tried using `substr` and just removing the first character, the problem are the "other" tokens... like `ukr` in Terra and all the others in e-Money... they also go through this function.

So right now I don't see any other solution :woman_shrugging:

Comment (jbibla) [reply src/scripts/common.js]: we should probably plan to store this in the db / api.

we want to get to a place where we don't need to update lunie frontend to add new networks.

(i think)

Comment (Bitcoinera) [reply src/scripts/common.js]: Yes, that would be the ideal

Comment (faboweb) [reply src/scripts/common.js]: #overengineering

Comment (Bitcoinera) [reply src/scripts/common.js]: hahaha. One thing is the ideal and another one the raw reality :laughing:

**Pull request merged**